### PR TITLE
No dev branch

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR prepare the `dev` branch deletion and needs to be merged at this moment.

### CODE REVIEW

File: docs/versionning.md [hunk 1]
```diff
@@ -6,42 +6,34 @@ UData uses [semantic versionning][semver] to publish its published releases.
...
-- the incoming new features (and their bug fixes)
-- the factoring
-- the dependencies upgrades
-- the bug fixes associated to these
+- the bug fixes and security upgrades associated to theses versions only
+- the backported bug fixes and security upgrades
 
-There is always at least one minor version between the `dev` and the `master branch`
-(*ie.* `1.0.2.dev` on `master` and `1.1.0.dev` on `dev`).
+Every version has a git tag `vX.Y.Z` and then when bug fixes need to be applied
```
Comment (abulte) [inline docs/versionning.md]: I'm not sure I understand this :-/
> then when bug fixes need to be applied out publishing new changes, there will be a version branch `vX.Y`

Comment (abulte) [reply docs/versionning.md]: Actually I think I do but it's not clear.

File: docs/versionning.md [hunk 2]
```diff
@@ -76,29 +68,31 @@ The step to perform a release are:
...
-(*ie.* `X.Y` when the tag is `X.Y.0`) and and publish it:
+We need to branch the master each time a minor version development cycle starts:
 
-```shell
-git checkout -b X.Y X.Y.0
-git push -u origin X.Y
-```
+- create a branch for the current minor version. **ex**: if the current master version is `1.1.8.dev`, create a `v1.1` branch with `git checkout -b v1.1`
+- publish the branch to the official repository: `git push -u origin vX.Y`
+- get th `master` branch back: `git checkout master`
```
Comment (abulte) [inline docs/versionning.md]: *the

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -49,8 +49,8 @@ basically you’ll have to:
 * commit incrementally with readable and detailed commit messages
 * add the change to the `CHANGELOG.md` file if appropriated
 * submit a pull-request against the right branch of this repository:
-    * `dev` for new features
-    * `master` for bugfixes (unless only present in `dev`)
+    * `master` for new features or bug fixes for the current state
+    * `vX.Y` for bugfixes or backports on previous versions
```
Comment (oncletom) [inline CONTRIBUTING.md]: Shall we refer to `vMAJOR.MINOR` to reflect the use of semver?

File: docs/versionning.md [hunk 4]
```diff
@@ -6,42 +6,33 @@ UData uses [semantic versionning][semver] to publish its published releases.
...
+- the backported bug fixes and security upgrades
 
-There is always at least one minor version between the `dev` and the `master branch`
-(*ie.* `1.0.2.dev` on `master` and `1.1.0.dev` on `dev`).
+Every minor version has a maintenance branch `vX.Y` and every version has a git tag `vX.Y.Z`.
 
-
-Every version has a git tag `X.Y.Z` and then when bug fixes need to be applied
-without publishing new changes, there will be a version branch `X.Y`
+Otherwise the version is `X.Y.Z.dev` (1.1.7.dev)
```
Comment (oncletom) [inline docs/versionning.md]: If we follow semver, should not it be `1.1.7-dev` instead? Cf. [hyphen separated suffix](http://semver.org/#spec-item-9).

Comment (noirbizarre) [reply docs/versionning.md]: It follows [Python versionning, developmental releases](https://www.python.org/dev/peps/pep-0440/#developmental-releases)

File: CONTRIBUTING.md (hunk 3 above)
Comment (noirbizarre) [reply CONTRIBUTING.md]: We don't follow semver (and we never had). I fixed the reference to semver to the rightful PEP 440

Comment (oncletom) [reply CONTRIBUTING.md]: Ah, this is stated in the `versionning.md` documentation file. Maybe worth referencing the right link and versionning scheme:

> UData uses [semantic versionning][semver] to publish its published releases.
> ...
> [semver]: http://semver.org/

**Pull request merged**